### PR TITLE
Basic manage profile

### PR DESCRIPTION
**Pull request by DenysVinegod**
# [Basic manage profile](https://trello.com/c/6WIjCqUu/30-basic-manage-profile-change-some-fields-avatar-etc-for-every-roles)

## Check list:
- [ ] Integration Tests
- [ ] Functional
- [x] Unit
- [ ] Factories
- [ ] Updated seed file

### CODE REVIEW

File: app/controllers/account/profile_controller.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class Account::ProfileController < Account::AccountsController
+  def edit
+    @user = current_user
+  end
+
+  def update
+    @user = current_user
```
Comment (manastyretskyi) [inline app/controllers/account/profile_controller.rb]: це линшє

File: app/views/account/profile/_form.html.slim [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  = simple_form_for user, url: account_profile_path(user) do |f|
+    .container
+        .mb-3.col-sm-5
+          = f.input :first_name, label: "First name:", input_html: { class: 'form__input' }
+        .mb-3.col-sm-5
+          = f.input :last_name, label: "Last name:", input_html: { class: 'form__input' }
+        .mb-3.col-sm-5
+          = f.input :email, label: "Email:", input_html: { class: 'form__input' }
+        .action
+          = f.submit "Send"
```
Comment (manastyretskyi) [inline app/views/account/profile/_form.html.slim]: \n

File: app/views/account/profile/edit.html.slim [hunk 3]
```diff
@@ -0,0 +1 @@
+= render  "account/profile/form", user: @user
```
Comment (manastyretskyi) [inline app/views/account/profile/edit.html.slim]: \n

**Review by manastyretskyi — CHANGES_REQUESTED** [review]

File: app/controllers/account/profile_controller.rb (hunk 1 above)
Comment (manastyretskyi) [reply app/controllers/account/profile_controller.rb]: просто current_user пиши там

**Review by manastyretskyi — APPROVED** [review]

**Review by mdotvasylenchuk — APPROVED** [review]

**Pull request merged**